### PR TITLE
chore: Post-migration cleanup

### PR DESCRIPTION
**Pull request by dnys1**
*Issue #, if available:*

*Description of changes:*
Post-migration clean up of missed items, and general clean up of pubspecs

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by HuiSF — APPROVED** [review]

File: packages/api/amplify_api_ios/example/pubspec.yaml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 name: amplify_api_ios_example
 description: Test bed for amplify_api_ios
-homepage: https://github.com/aws-amplify/amplify-flutter/tree/main/packages/api/amplify_api_ios
+version: 0.1.0
```
Comment (ragingsquirrel3) [inline packages/api/amplify_api_ios/example/pubspec.yaml:3]: same question here

File: packages/api/amplify_api_android/example/pubspec.yaml [hunk 2]
```diff
@@ -1,10 +1,10 @@
 name: amplify_api_android_example
 description: Test bed for amplify_api_android
-homepage: https://github.com/aws-amplify/amplify-flutter/tree/main/packages/api/amplify_api_android
-
-# The following line prevents the package from being accidentally published to
-# pub.dev using `flutter pub publish`. This is preferred for private packages.
-publish_to: "none" # Remove this line if you wish to publish to pub.dev
+version: 0.1.0
```
Comment (ragingsquirrel3) [inline packages/api/amplify_api_android/example/pubspec.yaml:3]: nit/question: Why do we need a version for example apps?

**Review by ragingsquirrel3 — APPROVED** [review]
approved w small, non-blocking question about versions in example apps

File: packages/api/amplify_api_android/example/pubspec.yaml (hunk 2 above)
Comment (dnys1) [reply packages/api/amplify_api_android/example/pubspec.yaml:3]: We technically don't. It's not required.

**Pull request merged**